ms2query/networking.py

```diff
@@ -70,3 +76,136 @@ def do_networking(query_id, matches, similarity_matrix):
     lib_ids = matches.index.tolist()
     network = add_library_connections(init_network, similarity_matrix, lib_ids)
     return network
+
+
+def draw_edges(network, pos, attribute, attr_cutoff, width_default,
+               style="solid", cmap=None, edge_labels=True):
+    """
+    Draw edges where width is determined by attribute score, if score > cutoff
+
+    Args:
+    -------
+    network: nx.Graph
```

Comment:
the typing should in the end be consistent in all functions (hence either in the docstring or in the function inputs). But feel free to ignore for now! I will simply add an issue for this for now.

Reply:
Yes good point. I am too used of doing it in the docstring haha. I will start doing it in the function inputs from now on and slowly move the existing docstring ones to the function inputs